UnitTests/MenuTests.cs

```diff
@@ -119,7 +119,7 @@ public void MenuOpening_MenuOpened_MenuClosing_Events ()
 				e.Action ();
 				Assert.Equal ("Copy", miAction);
 			};
-			menu.MenuClosing += () => {
+			menu.MenuClosing += (_) => {
```

Comment:
The fact you had to change this test, proves it is a breaking change!

Reply:
That's was a recent event implementation and I think it will not having a bigger impact.

Reply:
Done! https://github.com/migueldeicaza/gui.cs/pull/1615/commits/a2351b818105a7e45a3791befb382116298289c0
Improvements are more advantageous.